src/app/pipes/external-identifier.pipe.ts

```diff
@@ -0,0 +1,23 @@
+import { Pipe, PipeTransform } from '@angular/core';
+
+@Pipe({
+  name: 'externalIdentifier'
+})
+export class ExternalIdentifierPipe implements PipeTransform {
+
+  transform(externalId: string): string {
+    if (!externalId) {
+      return '';
+    } else {
+      const regexExp = /^[0-9a-fA-F]{8}\b-[0-9a-fA-F]{4}\b-[0-9a-fA-F]{4}\b-[0-9a-fA-F]{4}\b-[0-9a-fA-F]{12}$/gi;
+
+      if (regexExp.test(externalId)) {
```

Comment:
Is this necessary?

Reply:
Do we really need to truncate the external id?

Reply:
Yes, the Idea is to detect an externalId value with an UUID format and split this to get the latest part of the long value

Reply:
okay, but why? Dont we have enough space to show it? I dont think it will be straightforward, it is not the externalId, just the last part of it... no?

Reply:
<img width="1301" alt="Screenshot 2022-12-01 at 22 36 27" src="https://user-images.githubusercontent.com/44206706/205217709-db489f04-589f-4eeb-a416-d66ff675525b.png">